utils/navigator_tools/nodes/tf_to_gazebo.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+import roslib
+import rospy
+import math
+import tf
+import geometry_msgs.msg
+import numpy as np
+def main():
```

Comment:
2 spaces before function.
Also why have this as a function? Seems like it could all be in the `if __name__ == "__main__"` and be fine